scripts-dev/nuke-empty-communities-from-db.py

```diff
@@ -1,33 +1,49 @@
 #!/usr/bin/env python2
 
 from sys import argv
+from getopt import getopt
 import sqlite3
 import psycopg2
 
 if len(argv) < 3 or (argv[1] != "sqlite" and argv[1] != "postgresql") or (argv[1] == "postgresql" and len(argv) < 6):
```

Comment:
I think the `6` add the end of this line still makes options mandatory in practice.